<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRelativeForWorker</class>
 <widget class="QWidget" name="AddRelativeForWorker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加亲属</string>
  </property>
  <widget class="QLabel" name="worker_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择职工</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseWorkerBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="relative_gender_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeNameEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="relative_age_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="relative_name_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeAgeEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="relative_appellation_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>称谓</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeAppellationEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加亲属</string>
   </property>
  </widget>
  <widget class="QLabel" name="alertlabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="RelativeGendercomboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
